### DISCUSSION TITLE
Rounded corners not workng

### DISCUSSION THREAD

**Discussion by Crazysonoran** [Courseplay/Courseplay_FS22#3558] [Q&A] [answered]
I have been having problems getting my machines to do a sharp corner, even when the field has a 90' corner. Do I need to do anything else besides set the number of round corners to 0 and activate "Sharpen Headland corners"?  TIA

**Comment by Tensuko** [comment, marked as answer]
This is a known issue, we try to get a fix for this ready.

**Comment by rob230580** [comment]
Hi, just wondered if there is a solution to this? I just got 7.5 version and having the same issues. I cant seem to get the settings right, either all corners are rounded so lots of misses, or worker does turns even on the most gentle corners. If I have corners activated and increase rounded corners, it will choose the sharpest corner to round first....

**Reply by Provideniya** [reply]
Yeah, this is still a bit unfortunate behavior in otherwise good update.

Currently the options seems to be either to
-1 have green/red waypoint corners on like 90% of the long, gentle turns,
-2 have each Sharp Turn, including clear 90-degree (or so) corners be rounded on the headlands, leaves the outer part of corner untouched
-3 have route planned that recognizes both the Gentle Turns (which are planned correctly as yellow, following the field edge really well) and the Sharp Turns but plans them using the normal yellow waypoints, leaving big spots on the inside of the turns due vehicles turning normally and missing several waypoints as a result, especially with vehicles and tools with larger turn-circles.

Comparing the new routes to the ones saved from the previous versions, it looks like new version understands the field shapes better but just can't plan properly using that information. So in the end while the old one had some troubles with more complex shapes and sizes of the fields, it provided much more.. balanced approaches to the Routes.

**Reply by rob230580** [reply]
Yep totally agree, the old version was more efficient with corners but less accurate. I find now the best thing to do is activate corners to get the added accuracy but leave the number at five. You still get full coverage on the field when cultivating or planting but also less time taken on the gentle corners. On most field work it works well but will take a bit more time than the old version. 
The main problem is on grass work i find. Even if i have a forage wagon copyng the course of the krone mower, the corners will have a lot of misses with these settings.
